clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -283,8 +283,12 @@ describe("System management page", () => {
         expect(body.joint_controller.name).to.eql(controllerName);
       });
 
-      // add another privacy declaration
+      // Switch to the Data Uses tab
       cy.getByTestId("tab-Data uses").click();
+      // TODO: this is needed to dismiss the "Unsaved Changes" popup. Is that a bug?
+      cy.getByTestId("continue-btn").click()
```

Comment:
Mainly a question for @TheAndrewJackson: it might just be how we're stubbing the API calls here, but in this test the form is still "dirty" even after clicking save above. That surprised me and is either an issue with the testing stubs or a legit bug

Reply:
@TheAndrewJackson can you take a look at this? It's my only lingering TODO and I think it's a bug this test found...

Basically the steps here would be:
1. Go to https://fides-nightly.fides-staging.ethyca.com/
2. Navigate to "Data map" > "View systems", loads the default "System information" tab
3. Edit a system (in nightly right now, use "Test System" or anything)
4. Make a change to the system, hit "Save"
5. Navigate to the "Data use" tab
6. See the popup:

<img width="1658" alt="image" src="https://user-images.githubusercontent.com/1834295/223507219-f3de35ba-01fd-49a3-95bf-b0a76963be75.png">

This doesn't seem right- I already saved the changes.

Reply:
Opened https://github.com/ethyca/fides/issues/2788 for this, so I can tidy up this PR. Thanks!